plugins/git/README.md

```diff
@@ -89,7 +89,7 @@ plugins=(... git)
 | `gdnolock`             | `git diff $@ ":(exclude)package-lock.json" ":(exclude)\*.lock"`                                                                 |
 | `gdt`                  | `git diff-tree --no-commit-id --name-only -r`                                                                                   |
 | `gf`                   | `git fetch`                                                                                                                     |
-| `gfa`                  | `git fetch --all --tags --prune`                                                                                                       |
```

Comment:
I'm failing to see what's the change here?

Reply:
The table was reformatted as a style check such that the columns are the same size everywhere (so the document is easy to read even as raw unrendered Markdown), and this line was too long. There were exactly 7 space characters deleted before the last `|`:

![Screenshot_20240811_104536](https://github.com/user-attachments/assets/07dcf6de-511a-4ea0-91ad-eea17742ba56)

Reply:
Ah thanks, I could not spot that in the diff.